tests/product/tests/data/aws/aws_iam_test_cases.py

```diff
@@ -230,6 +233,35 @@
     "1.15 Permissions through groups: inline policy, expect: failed": cis_aws_iam_1_15_fail,
 }
 
+cis_aws_iam_1_16 = {
+    "1.16 Built-in EC2 policy, expect: passed": EksAwsServiceCase(
+        rule_tag=CIS_1_16,
+        case_identifier="AmazonEC2FullAccess",
+        expected=RULE_PASS_STATUS,
+    ),
```

Comment:
@orestisfl, this test case (1.16) fails -> expected passed retrieved failed. Please check again if expected result is correct.

Reply:
Should be fixed by https://github.com/elastic/csp-security-policies/pull/228

Reply:
@gurevichdmitry please check again